src/styles/globals.css

```
@@ -173,10 +240,9 @@
   --radius-4xl: calc(var(--radius) + 16px);
 }
 
-@import './themes/zinc.css';
-@import './themes/catppuccin.css';
-@import './themes/t3chat.css';
-@import './themes/claude.css';
+@import 'tw-animate-css';
```

Biome (2.4.4)

[error] This `@import` is in the wrong position.

(lint/correctness/noInvalidPositionAtImportRule)